From 16144883e180994a2833646e57764434b2802404 Mon Sep 17 00:00:00 2001
From: Yongsheng Chen <yongsheng.chen@intel.com>
Date: Fri, 5 Oct 2018 12:42:18 -0700
Subject: [PATCH 2293/2345] media: intel-ipu4: ox03a10: Disable normalization.

Message for Open Source:
Disable normalization in sensor.

Message for Internal:
Disable normalization in sensor.

Change-Id: Idc3c4e1036e645bb72d71b677f52dcdea77f45cf
Signed-off-by: Chen, Yongsheng <yongsheng.chen@intel.com>
Signed-off-by: Meng Wei <wei.meng@intel.com>
---
 drivers/media/i2c/crlmodule/crl_ox03a10_configuration.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/media/i2c/crlmodule/crl_ox03a10_configuration.h b/drivers/media/i2c/crlmodule/crl_ox03a10_configuration.h
index 92d4fab..4f4a853 100644
--- a/drivers/media/i2c/crlmodule/crl_ox03a10_configuration.h
+++ b/drivers/media/i2c/crlmodule/crl_ox03a10_configuration.h
@@ -413,7 +413,7 @@ static struct crl_register_write_rep ox03a10_1920_1088_12DCG[] = {
 	{ 0x54a4, CRL_REG_LEN_08BIT, 0x04 },
 	{ 0x54a5, CRL_REG_LEN_08BIT, 0x00 },
 	{ 0x54a7, CRL_REG_LEN_08BIT, 0x00 },
-	{ 0x5800, CRL_REG_LEN_08BIT, 0x39 },
+	{ 0x5800, CRL_REG_LEN_08BIT, 0x31 },
 	{ 0x5801, CRL_REG_LEN_08BIT, 0x03 },
 	{ 0x5804, CRL_REG_LEN_08BIT, 0x00 },
 	{ 0x5805, CRL_REG_LEN_08BIT, 0x40 },
@@ -867,7 +867,7 @@ static struct crl_register_write_rep ox03a10_1920_1088_12DCG_12VS[] = {
 	{ 0x54a4, CRL_REG_LEN_08BIT, 0x04 },
 	{ 0x54a5, CRL_REG_LEN_08BIT, 0x00 },
 	{ 0x54a7, CRL_REG_LEN_08BIT, 0x00 },
-	{ 0x5800, CRL_REG_LEN_08BIT, 0x39 },
+	{ 0x5800, CRL_REG_LEN_08BIT, 0x31 },
 	{ 0x5801, CRL_REG_LEN_08BIT, 0x03 },
 	{ 0x5804, CRL_REG_LEN_08BIT, 0x00 },
 	{ 0x5805, CRL_REG_LEN_08BIT, 0x40 },
-- 
2.7.4

